pkg/web_app/lib/src/screenshot_carousel.dart

```diff
@@ -57,7 +57,7 @@ void _setEventForScreenshot() {
   for (final thumbnail in thumbnails) {
     thumbnail.onClick.listen((event) {
       showElement(carousel);
-      images = thumbnail.attributes['data']!.split(',');
+      images = thumbnail.attributes['thumbnail-data']!.split(',');
```

Comment:
I think this should rather be `data-<x>` attribute, that can be accessed via `Element.dataset['x']`.

Reply:
Definitely! :-) Done.